### PR TITLE
add leaflet.freedraw v1.2.2 with npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6746 
@dannyxx001 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Wildhoney/Leaflet.FreeDraw
 * Official website (optional, not the repository): https://freedraw.herokuapp.com/
 * NPM package url (optional): https://www.npmjs.com/package/leaflet.freedraw
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 222
   - Count of watchers: 18
   - Count of forks: 38
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 125
   - Downloads in the last month: 2640

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #6746, cc @Wildhoney

### CODE REVIEW

File: ajax/libs/leaflet.freedraw/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "homepage": "https://freedraw.herokuapp.com",
+  "dependencies": {
+    "express": "^4.9.0"
+  },
+  "npmName": "leaflet.freedraw",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "leaflet.freedraw*"
```
Comment (PeterDaveHello) [inline ajax/libs/leaflet.freedraw/package.json]: Why not `"**/*"` ?

File: ajax/libs/leaflet.freedraw/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "keywords": [
+    "leaflet",
+    "freedraw",
+    "polygon",
+    "drawing",
+    "freehand"
+  ],
+  "author": "Adam Timberlake <adam.timberlake@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://freedraw.herokuapp.com",
```
Comment (PeterDaveHello) [inline ajax/libs/leaflet.freedraw/package.json]: I can not open this link, can you?

**Comment by kennynaoh** [comment]
@dannyxx001 
I have fixed the ```npmFileMap```.
Please help me review it again.
Thanks.

**Comment by dannyxx001** [comment]
I think this PR is OK. @PeterDaveHello please help us review again, thanks.

**Pull request merged**